### PR TITLE
Minor fixes to the 'getting started' page

### PR DESCRIPTION
**Pull request by biggianteye**
- Use consistent visibility for some fields.
- Fixing path of yaml file.
- Minor grammar changes.

### CODE REVIEW

File: docs/en/tutorials/getting-started.rst [hunk 1]
```diff
@@ -717,8 +717,8 @@ the bi-directional reference:
     {
         // ... (previous code)
 
-        private $reportedBugs = null;
-        private $assignedBugs = null;
+        protected $reportedBugs = null;
```
Comment (Ocramius) [inline docs/en/tutorials/getting-started.rst]: Should stay `private`

File: docs/en/tutorials/getting-started.rst [hunk 2]
```diff
@@ -717,8 +717,8 @@ the bi-directional reference:
...
+        protected $assignedBugs = null;
```
Comment (Ocramius) [inline docs/en/tutorials/getting-started.rst]: Should stay `private`

**Review by Ocramius — CHANGES_REQUESTED** [review]

File: docs/en/tutorials/getting-started.rst (hunk 1 above)
Comment (biggianteye) [inline docs/en/tutorials/getting-started.rst]: Could you explain why? All other instance variables are marked as `protected`. Why should these two be `private`?

More importantly, they were declared as `protected` when first added to the `User` class. If there is a legitimate need for the change to the visibility, let me know and I'll incorporate it into the section where the visibility changes.

Comment (Ocramius) [inline docs/en/tutorials/getting-started.rst]: I'd rather have everything as `private` around the code, but if you think that `protected` is better for consistency (for now), I can merge the PR as-is.

Should generally strive for `private` in future.

Comment (biggianteye) [inline docs/en/tutorials/getting-started.rst]: I'm not trying to advocate one over the other (though if I were, I'd go for private). I'm just trying to remove any confusion caused by modifiers changing half way through introductory documentation without any explanation.

Comment (Ocramius) [inline docs/en/tutorials/getting-started.rst]: Good, merging as-is then :-)

File: docs/en/tutorials/getting-started.rst (hunk 2 above)
Comment (biggianteye) [inline docs/en/tutorials/getting-started.rst]: 👍

File: docs/en/tutorials/getting-started.rst (hunk 1 above)
Comment (biggianteye) [inline docs/en/tutorials/getting-started.rst]: 👍

**Pull request merged**